src/org/traccar/protocol/RitiProtocolDecoder.java

```diff
@@ -66,15 +66,15 @@ protected Object decode(
         position.setDeviceId(deviceSession.getDeviceId());
 
         position.set("mode", buf.readUnsignedByte());
-        position.set("command", buf.readUnsignedByte());
-        position.set(Position.KEY_POWER, buf.readUnsignedShort());
+        position.set(Position.KEY_COMMAND, buf.readUnsignedByte());
+        position.set(Position.KEY_POWER, buf.readUnsignedShort());  // mV
 
-        buf.skipBytes(5);
-        buf.readUnsignedShort();
-        buf.readUnsignedShort();
+        buf.skipBytes(5);  // status
+        buf.readUnsignedShort();  // idleCount
+        buf.readUnsignedShort();  // idleTime in seconds
 
-        position.set(Position.KEY_DISTANCE, buf.readUnsignedInt());
-        position.set(Position.KEY_TRIP_ODOMETER, buf.readUnsignedInt());
+        position.set(Position.KEY_DISTANCE, buf.readUnsignedInt());  // meters
```

Comment:
We should use `KEY_ODOMETER` instead of `KEY_DISTANCE`. Also, there is no need to put "meters" in the comment. It would always be in meters. You can put comment in the `Position` class.